### DISCUSSION TITLE
Runway statamic 4 update

### DISCUSSION THREAD

**Discussion by igmltd** [statamic-rad-pack/runway#281] [Help]
Hi Duncan,

I updated to Statamic 4, i had to remove runway, i'm now i'm trying to reinstall, however i'm getting this issue:

Problem 1
    - Root composer.json requires doublethreedigital/runway ^5.0 -> satisfiable by doublethreedigital/runway[v5.0.0, ..., 5.x-dev].
    - doublethreedigital/runway[v5.0.0, ..., 5.x-dev] require php ^8.2 -> your php version (8.1.15) does not satisfy that requirement.
    

is php 8.2 required for runway???

NOTE:
im using 'composer require doublethreedigital/runway ^5.0' to install 

when i use just 'composer require doublethreedigital/runway' as on the docs, it tries to install runway v 4.1, which fails.

    Thanks

**Comment by duncanmcclean** [comment]
Yes, Runway requires PHP 8.2.

**Reply by igmltd** [reply]
Thanks - 8.2 might be an issue for me at the moment to install, would it not work at all with 8.1? 

or is there version of runway that would work with statamic 4 and 8.1?

thanks

**Reply by duncanmcclean** [reply]
Not at the moment. Why can't you use 8.2?

**Reply by igmltd** [reply]
> Not at the moment. Why can't you use 8.2?

Updating php on the server will be an issue as only 8.1 is currently available : (

**Reply by igmltd** [reply]
Just an update

i used:

`composer require doublethreedigital/runway ^5.0 --ignore-platform-req=php`

everything i'm using runway for  (i.e template tags to display db data) seems to work just fine on php 8.1

is there anything i should be aware of?

thanks

**Reply by duncanmcclean** [reply]
I'll consider adding back PHP 8.1 support to the addon.

**Comment by duncanmcclean** [comment]
Runway [v5.0.8](https://github.com/duncanmcclean/runway/releases/tag/v5.0.8) brings back support for PHP 8.1
